src/main/scala/io/iohk/ethereum/domain/Account.scala

```diff
@@ -1,21 +1,26 @@
 package io.iohk.ethereum.domain
 
 import akka.util.ByteString
-import org.spongycastle.util.encoders.Hex
+
+object Account {
+  val Empty = Account(0, 0, ByteString.empty, ByteString.empty)
```

Comment:
@adamsmo @LukasGasior1 

```
private val EmptyAccountStorageHash = ByteString(Hex.decode("56e81f171bcc55a6ff8345e692c0f86e5b48e01b996cadc001622fb5e363b421"))
private val EmptyAccountEvmCodeHash = ByteString(Hex.decode("c5d2460186f7233c927e7db2dcc703c0e500b653ca82273b7bfad8045d85a470"))
```

I know that `EmptyAccountEvmCodeHash` is simply `kec256(ByteString())` (and that's how I'm going to write it) and it's direct consequence of YP section 4.1 (p.3).

Were did we get the number for `EmptyAccountStorageHash`? How is it calculated?